src/Nethermind/Nethermind.Evm/VirtualMachine.cs

```diff
@@ -439,8 +439,8 @@ public TransactionSubstate Run<TTracingActions>(EvmState state, IWorldState worl
 
                 if (txTracer.IsTracingInstructions)
                 {
-                    txTracer.ReportOperationError(ex is EvmException evmException ? evmException.ExceptionType : EvmExceptionType.Other);
                     txTracer.ReportOperationRemainingGas(0);
+                    txTracer.ReportOperationError(ex is EvmException evmException ? evmException.ExceptionType : EvmExceptionType.Other);
```

Comment:
Why are we reordering the operations?

Reply:
Now they are always same order, this shoulnd't break anything in existing chains but help with ordering in javascript one